frontend/src/util/LiveMessageManager.js

```diff
@@ -75,7 +76,11 @@ export class LiveMessageManager {
     this.closeEventSourceConnection()
 
     this.eventSource = new EventSource(
-      `${url}/users/${this.userId}/live_messages`,
+      `${url}/users/${this.userId}/live_messages` + (
+        this.lastEventId
+          ? `?after_event_id=${this.lastEventId}`
+          : ''
+        ),
```

Comment:
Wouldn't it be more readable by writing like this?

```javascript
let afterEventFilter = ''
if this.lastEventId:
  afterEventFilter = '?after_event_id="${this.lastEventId}`
this.eventSource = new EventSource(
   `${url}/users/${this.userId}/live_messages${afterEventFilter}`
...
```

Reply:
I don't have a strong opinion here, I am going to go for your version.